### DISCUSSION TITLE
New Catalog page

### DISCUSSION THREAD

**Discussion by Shnitselon** [QualiTorque/qualitorque.github.io#186] [Announcements]
We're happy to announce our new **Catalog** page. Now, your end-users have a place to browse and launch the different blueprints that are available for them. This addition goes hand in hand with [blueprint labels](https://github.com/QualiTorque/qualitorque.github.io/discussions/184), which allow you to customize the look of the catalog's blueprints.
![image](https://user-images.githubusercontent.com/20701198/208375962-c1e6d510-dea9-4f0c-9fb2-ebbb1ada70d6.png)

**Comment by TomerAdmon** [comment]
Brilliant 😍😍
